### PR TITLE
Fix browser bar missing focus description

### PR DESCRIPTION
**Pull request by msmanoj**
## Type

- [x] Bug fix
- [ ] New feature
- [ ] Improvement to existing feature

## Related JIRA Issue(s)
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-192

## Importance
June-alpha hotfix

## Description
Fixed the missing focus description in the BrowserBar and created the type BrowserChrLocation.

## Views affected
Brower -> BrowserBar -> BrowserInfo

### CODE REVIEW

File: src/ensembl/src/content/app/browser/browserState.ts [hunk 1]
```diff
@@ -23,6 +23,8 @@ export type BrowserNavStates = [
 
 export type ChrLocation = [string, number, number];
 
+export type BrowserChrLocation = { [ensObjectId: string]: ChrLocation };
```
Comment (azangru) [inline src/ensembl/src/content/app/browser/browserState.ts]: the word "browser" in `BrowserChrLocation` doesn't really help much, does it?
We used to have a single chromosome location; now we have multiple chromosome locations keyed by genomeId; does it mean that the type is `ChrLocations`? Which probably means that the `chrLocation` field of the state becomes `chrLocations`?

Also, I don't know whether it's better to have a state that looks like this:

```
{
  chrLocations: {
    id1: [1, 234, 345]
    id2: [1, 334, 445]
  },
  defaultChrLocations: {
    id1: [1, 234, 345]
    id2: [1, 334, 445]
  }
}
```

or:

```
{
 id1: {
     chrLocation: [1, 234, 345],
     defaultChrLocation: [1, 234, 345]
  },
 id2: {
     chrLocation: [1, 334, 445],
     defaultChrLocation: [1, 334, 445]
  },
}
```

It's all so confusing. Can you offer any suggestions, @ens-ds23 ?

File: src/ensembl/src/content/app/browser/Browser.tsx [hunk 2]
```diff
@@ -197,11 +199,9 @@ export const Browser: FunctionComponent<BrowserProps> = (
   }, [props.browserQueryParams.focus]);
 
   useEffect(() => {
-    const chrLocationForGenome = props.chrLocation[props.activeGenomeId] || [
-      '',
-      0,
-      0
-    ];
+    const chrLocationForGenome = props.chrLocation[
+      props.activeEnsObjectId[props.activeGenomeId]
```
Comment (azangru) [inline src/ensembl/src/content/app/browser/Browser.tsx]: How did we get to the point when a prop called something-something-id is in fact a hash of ids? Should we be passing this whole hash to the Browser component? Should we not extract the relevant id in the mapStateToProps function?

(Don't know whether this grumble belongs to this PR or to a separate refactoring effort, but it just reads wrong.)

File: src/ensembl/src/content/app/browser/Browser.tsx [hunk 3]
```diff
@@ -197,11 +199,9 @@ export const Browser: FunctionComponent<BrowserProps> = (
...
+    ] || ['', 0, 0];
```
Comment (azangru) [inline src/ensembl/src/content/app/browser/Browser.tsx]: Is there a point in having a default chromosome location ['', 0, 0] from the genome browser perspective, @ens-ds23?

File: src/ensembl/src/content/app/browser/browserState.ts (hunk 1 above)
Comment (msmanoj) [reply src/ensembl/src/content/app/browser/browserState.ts]: `ChrLocations` and `ChrLocation`? Wouldn't it be hard to distinguish?

File: src/ensembl/src/content/app/browser/Browser.tsx (hunk 2 above)
Comment (msmanoj) [reply src/ensembl/src/content/app/browser/Browser.tsx]: How about renaming it to `activeEnsObjects` that will have hash of ids?

Also, I would prefer to get this refactoring done in a separate PR.

File: src/ensembl/src/content/app/browser/browserState.ts (hunk 1 above)
Comment (azangru) [reply src/ensembl/src/content/app/browser/browserState.ts]: Don't know. But the name `BrowserChrLocation` doesn't indicate in any way how this is different from a `ChrLocation` (which is also used by genome browser, so, in a sense, it is also a `BrowserChrLocation`).

**Pull request merged**